content/ko/docs/community/related.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "관련 프로젝트와 문서"
+description: "커뮤니티에서 제공하는 서드파티 도구, 플러그인 및 문서"
+weight: 3
+aliases: ["/docs/related/"]
```

Comment:
Please remove this line. Aliases to English should not appear in translations, as that will redirect readers from English to a translation.

Reply:
Yes, I  will remove aliases as well as also in future translations.